doc/source/io.rst

```diff
@@ -241,25 +255,56 @@ footprints while the black grids illustrate the ``DataArray`` chunks.
 Writing DataArrays to file
 --------------------------
 
-GeoWombat's I/O can be accessed through the :func:`to_vrt` and :func:`to_raster` functions. These functions use
-Rasterio's :func:`write` and Dask.array :func:`store` functions as I/O backends. In the examples below,
-``src`` is an ``xarray.DataArray`` with the necessary transform information to write to an image file.
+GeoWombat's file writing can be accessed through the :func:`to_vrt`, :func:`to_raster`,
+and :func:`save` functions. These functions use Rasterio's :func:`write` and ``Dask.array``
+:func:`store` functions as I/O backends. In the examples below, ``src`` is an ``xarray.DataArray``
+with the necessary transform and coordinate reference system (CRS) information to write to an
+image file.
 
-Write to a VRT file.
+.. note::
+
+    Should I use :func:`to_raster` or :func:`save` when writing a raster file? First, a bit of
```

Comment:
This is great but when you refer to `Dask` but not `to_raster` or `save` it gets unclear about which method you are describing. I am going to insert some questions in order to help you clarify. 


  The :func:`to_raster` can be used on data of any size, but comes with its own overhead. For example, when working with arrays that fit into memory, such as a standard satellite scene, ``Dask`` **with dask are you refering to to_raster or save?** works quite well. To give an example, instead of slicing a ``DataArray`` chunk and writing/computing that chunk (i.e., :func:`to_raster` approach), we can also compute the entire  ``DataArray`` using ``Dask`` **with the :func:`save` method?** and let ``Dask`` handle the concurrency. This is where :func:`save`comes in to play. The ``geowombat.save`` method (or also ``DataArray.gw.save``) submits the data  to ``Dask.array.store`` and each chunk is written to file using ``rasterio``.